app/controllers/lookup_keys_controller.rb

```diff
@@ -34,4 +33,14 @@ def destroy
   def controller_permission
     'external_variables'
   end
+
+  def find_resource
```

Comment:
This is defining both `@lookup_key` and  `@variable_lookup_key` or `@puppetclass_lookup_key`. How about overriding `resource_name` to `lookup_key`?